### PR TITLE
Minor cleanups

### PR DESCRIPTION
**Pull request by SeanFeldman**
## Proposed Changes

Listed in the commits

### CODE REVIEW

File: src/Promitor.Core.Scraping/Scraper.cs [hunk 1]
```diff
@@ -83,7 +83,7 @@ public async Task ScrapeAsync(MetricDefinition metricDefinition)
 
                 _logger.LogInformation("Found value '{MetricValue}' for metric '{MetricName}' with aggregation interval '{AggregationInterval}'", foundMetricValue, metricDefinition.Name, aggregationInterval);
 
-                var metricsTimestampFeatureFlag = FeatureFlag.IsActive("METRICSTIMESTAMP", defaultFlagState: true);
```
Comment (tomkerkhove) [inline src/Promitor.Core.Scraping/Scraper.cs]: Actually this is intentional for readability

Comment (SeanFeldman) [reply src/Promitor.Core.Scraping/Scraper.cs]: addressed

**Comment by tomkerkhove** [comment]
Thanks @SeanFeldman 	!

**Comment by SeanFeldman** [comment]
❤️

**Pull request merged**